### PR TITLE
Increment decimal

### LINKED ISSUE #127: Create "Increment a decimal-coded number" problem

### PR DESCRIPTION
**Pull request by lumig242**
Create a new question "Increment decimal".

Fix #127

### CODE REVIEW

**Comment by eyurko** [comment]
Let me know when @arbiter527 has finished reviewing and I'll take a look.

File: assets/questions/plusone.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+/**
+ * @fileoverview Question data for Reverse Words.
+ */
+
+globalData.questions['plusone'] = {  // eslint-disable-line dot-notation
+  title: 'Increment a decimal-coded number',
+  starterCode: {
+    python:
+`def increment(array):
```
Comment (arbiter527) [inline assets/questions/plusone.js]: This input is an array or string?

File: assets/questions/plusone.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    return '0'
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def foo(cls, array):
```
Comment (arbiter527) [inline assets/questions/plusone.js]: What is foo? Should add some buggy functions?

File: assets/questions/plusone.js (hunk 1 above)
Comment (lumig242) [inline assets/questions/plusone.js]: String. Changed

File: assets/questions/plusone.js (hunk 2 above)
Comment (lumig242) [inline assets/questions/plusone.js]: Removed this function.

File: assets/question_sets/strings.js [hunk 3]
```diff
@@ -26,5 +26,5 @@ globalData.questionSets['strings'] = {  // eslint-disable-line dot-notation
       "I will provide you with feedback."
     ].join('')
   ],
-  questionIds: ['reverseWords', 'parens', 'i18n', 'rle']
+  questionIds: ['reverseWords', 'parens', 'i18n', 'rle', 'plusone']
```
Comment (eyurko) [inline assets/question_sets/strings.js]: As I mentioned in this morning's email, you shouldn't try to commit with your function name in strings.js.

File: assets/questions/plusone.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Reverse Words.
+ */
+
+globalData.questions['plusone'] = {  // eslint-disable-line dot-notation
```
Comment (eyurko) [inline assets/questions/plusone.js]: increment, probably more than plus one. If it were plus one, it should be plusOne, anyways.

File: assets/questions/plusone.js [hunk 5]
```diff
@@ -27,7 +27,9 @@ globalData.questions['plusone'] = {  // eslint-disable-line dot-notation
   auxiliaryCode: {
     python:
 `class AuxiliaryCode(object):
-    pass
+    @classmethod
+    def foo(cls, array):
```
Comment (eyurko) [inline assets/questions/plusone.js]: This isn't an actual buggy test, though. Please write code to support a buggyOutputTest.

File: assets/questions/plusone.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+`class AuxiliaryCode(object):
+    @classmethod
+    def foo(cls, array):
+        return none
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        'Given a number represented as a string, increment it.'
```
Comment (eyurko) [inline assets/questions/plusone.js]: Write a function increment that takes in a number represented as a string and increments it. This function should return the incremented number.

**Comment by eyurko** [comment]
It looks like this functionality is duplicated in #175. Which of your groups claimed the question first?

**Comment by lumig242** [comment]
Oops. But I think we created the issue first at  #127 .  And #175 is linked to #133 .

**Comment by lumig242** [comment]
@eyurko please take another look at the changes :)

**Comment by eyurko** [comment]
Also, you should check to make sure the Travis build has succeeded before you LGTM -- currently, you're failing some linter checks.

**Comment by lumig242** [comment]
@eyurko @arbiter527  Let's close this pr. We have merged our test cases to #175

**Pull request closed without merging**